library/lib/build.js

```diff
@@ -94,6 +95,8 @@ module.exports = function build({ watch }) {
 
   const srcDir = path.resolve(process.cwd(), 'src')
 
+  const yarnWorkspaceDir = findYarnWorkspaceRoot(path.resolve(__dirname, '..'))
```

Comment:
This might not be the best way to do it. `__dirname` refers to the location of the current module. So if I understand your intention you probably need the `cwd` directory.

On top of that, workspaces allow packages to be nested deeper, in that case `..` would not be sufficient.

Seems they are working on proper support. So until then we could probably live with the limitation of `..`.

https://github.com/liady/webpack-node-externals/pull/45
https://github.com/liady/webpack-node-externals/issues/39#issuecomment-367778301

Reply:
The `find-yarn-workspace-root` expects a directory with a `package.json` to be able to match this package-name higher up in the directory structure. Since this file (`build.js`) lives in the `lib/`-directory I had to point it one directory higher. If I'm correct it won't break with deeper nested packages.

Reply:
Oh, haha, I missed `findYarnWorkspaceRoot` for some strange reason. That would fix the issue of deeper nesting. I wonder however if we should start at `__dirname/..` or `cwd`.